README.md

```diff
@@ -15,20 +15,29 @@ There is also hardware available for hackers to check out and hack with.
 This website is developed with [Gatsby](https://www.gatsbyjs.org/) and
 [Material UI](https://material-ui.com/), and is a work-in-progress being
 developed by ACM Hack’s Dev Team: 
-- Alex Xia [(@khxia)](https://github.com/khxia)
+- Aazel Tan [(@aazeltan)](https://github.com/aazeltan)
```

Comment:
Nice work! Typically, we have the current dev team leads (Jakob, Katelyn, Maggie) at the top of the list. Members thereafter are typically alphabetized, too! Do you mind changing the order of the contributors, thanks Sam!

Reply:
Yes. See commit 46f76f2